src/utils/environment.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+import tum_esm_utils
+
+
+CONFIG_DIR_KEY = "ERP_CONFIG_DIR"
```

Comment:
Please hardcode this into the function

Reply:
As this variable is used by both functions in the file, I don't think it should be a local variable, if that's what you meant?